rosserial_server/include/rosserial_server/utils.h

```diff
@@ -0,0 +1,35 @@
+/******************************************************************************
+ *  Copyright (c) 2024, Intermodalics BVBA                                    *
+ *  All rights reserved.                                                      *
+ ******************************************************************************/
+
+#ifndef ROSSERIAL_SERVER_UTILS_H
+#define ROSSERIAL_SERVER_UTILS_H
+
+#include <filesystem>
```

Comment:
```suggestion
```

Not required anymore, and neither is setting `CMAKE_CXX_STANDARD` explicitly in `CMakeLists.txt`.